openhtf/exe/phase_executor.py

```diff
@@ -55,6 +57,16 @@
 class InvalidPhaseResultError(Exception):
   """Raised when a PhaseOutcome is created with an invalid phase result."""
 
+class ExceptionOutcome(collections.namedtuple('ExceptionOutcome', ['exc_type','exc_val','exc_tb'])):
+
+  def _asdict(self):
+    return_dict = {}
```

Comment:
it's kind of a personal style thing, but i usually like inline dict constructs rather than by-key, so:
```
return {
    'exc_type': str(self.exc_type),
    'exc_val': self.exc_val,
    'exc_tb': ''.join(traceback.format_exception(*self)),
}